### PR TITLE
Added Sarika's Profile

### LINKED ISSUE #317: Add Sarika profile
**What needs doing?**
Image, 3 words and short text + optional social links.

### PR DESCRIPTION
**Pull request by SarikaWirtz**
Fixes #317

## Description

- Sarika Profile card with image

### CODE REVIEW

File: src/data/tribers/sarika.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: 'Sarika'
+words:
+  - Front-end web development
+  - Learning
+  - Animals
+image: images/sarika.jpg
+story: stories/sarika.md
```
Comment (katjam) [inline src/data/tribers/sarika.md]: You won't need this part. The stories are the ones on the front page of the site with their text stored in the `/data/tribers/stories` folder. 

For now we have only written for 3 people in the company about journey to get here.

File: src/data/tribers/sarika.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+---
+
+I completed my bachelor degree in computer science from India and then started a job as a SEO executive for few years and then I got interested in front-end development. I worked as a front end developer for 1.5 years and then took a long break (Maternity). Now i want start once again with my carrer in front-end development but i was so confuse where to start with. Then Neontribe gave me a chance to start again. At Neontribe i will work as a 'Frontend dev intern'.
```
Comment (katjam) [inline src/data/tribers/sarika.md]: 2 different spellings of front end front-end. Pick either and make them the same.

Comment (katjam) [reply src/data/tribers/sarika.md]: Uppercase the letter `i` even when it is middle of a sentence.

Comment (katjam) [reply src/data/tribers/sarika.md]: Change carrer to career

File: src/data/tribers/sarika.md (hunk 1 above)
Comment (Nikomus) [reply src/data/tribers/sarika.md]: We have a ticket for more people to be featured on the front page, but not yet 🙂 
https://github.com/neontribe/www/issues/270

File: src/data/tribers/sarika.md (hunk 2 above)
Comment (SarikaWirtz) [reply src/data/tribers/sarika.md]: OMG I did so silly mistakes …. thank you for correcting me katja

File: src/data/tribers/sarika.md (hunk 1 above)
Comment (SarikaWirtz) [reply src/data/tribers/sarika.md]: thank you for letting me know

**Comment by RpprRoger** [comment]
Thanks for you PR @SarikaWirtz, it seems like these things we fixed in #318, so I'll close this one and review the other

**Pull request closed without merging**